packages/communicator-email-ses/index.ts

```diff
@@ -0,0 +1,152 @@
+import {Communicator, CommunicatorContext, Configurable, InitContext} from '@dbos-inc/dbos-sdk';
+
+import { SESv2, SendEmailCommand } from '@aws-sdk/client-sesv2';
+import { AWSServiceConfig, getAWSConfigForService, getAWSConfigs } from '@dbos-inc/aws-config';
+
+interface SESConfig{
+    awscfgname?: string,
+    awscfg?: AWSServiceConfig,
+}
+
+@Configurable()
+class SendEmailCommunicator
+{
+    static AWS_SES_CONFIGURATIONS = 'aws_ses_configurations';
+    static async initConfiguration(ctx: InitContext, arg: SESConfig) {
+        // Get the config and call the validation
+        getAWSConfigs(ctx, SendEmailCommunicator.AWS_SES_CONFIGURATIONS);
+        if (!arg.awscfg) {
+            arg.awscfg = getAWSConfigForService(ctx, this.AWS_SES_CONFIGURATIONS, arg.awscfgname ?? "");
+        }
+        return Promise.resolve();
```

Comment:
Since we're already in an async context, why are we returning a resolved promise?

Reply:
My understanding is that is our new way of making the linter happy/happier without a comment saying "yeah I know this method has no business being async".